src/components/TopBar/TopBarStyles.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+
+const bottomLineHeight = 1;
+const topBarHeight = 56;
+
+export const StyledTopBarWrapper = styled.nav`
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: ${topBarHeight + bottomLineHeight}px;
+`;
+
+export const StyledTopBar = styled.div`
+  width: 100%;
+  height: ${topBarHeight}px;
+  padding: 16px;
+  background-color: #2a2a2a;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.12), 0px 2px 4px rgba(0, 0, 0, 0.14),
+    0px 1px 2px rgba(0, 0, 0, 0.2);
+`;
+
+export const StyledHeader = styled.header`
```

Comment:
What do you mean by "header"?
Isn't the whole top bar a "header" in some way? :)